io.gdevelop.ide.yml

```diff
@@ -0,0 +1,54 @@
+# This is the Flatpak manifest for GDevelop.
+app-id: io.gdevelop.ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --filesystem=xdg-run/pipewire-0
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=ca.desrt.dconf
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.Notifications
+  - --env=XDG_CURRENT_DESKTOP=Unity
+modules:
+  - name: gdevelop-ide
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://gdevelop-releases.s3.amazonaws.com/master/commit/50326e4d2147e0b7efca02ef062dba97831cd3d7/gdevelop_5.0.130_amd64.deb
+        sha256: 59a6812d082e25b7ca7dd54a31482bb5187f3542a6f99d3dc9ffee8895b6cde6
+      - type: file
+        path: io.gdevelop.ide.appdata.xml
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh "/app/GDevelop 5/gdevelop" "$@"
+
+    build-commands:
+      - ar x gdevelop_5.0.130_amd64.deb
+      - tar -xf data.tar.xz
+      - cp -r ./opt/* /app/
+      - cp -r ./usr/share/* /app/share/
+      - ln -s "/app/GDevelop 5/gdevelop" /app/bin
+      - install -Dm644 io.gdevelop.ide.appdata.xml -t /app/share/metainfo
+      - mv ${FLATPAK_DEST}/share/applications/gdevelop.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key Exec --set-value "run.sh gdevelop %U" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key Icon --set-value "io.gdevelop.ide" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - rm -rf ${FLATPAK_DEST}/share/icons/hicolor/1024x1024/
+      - find ${FLATPAK_DEST}/share/icons -type f -name *.png -exec rename gdevelop.png ${FLATPAK_ID}.png '{}' \;
```

Comment:
there is a `rename-icon` that should do that automatically for you (including the part where you have to set the icon name in the desktop file.